be/src/util/bitmap_value.h

```diff
@@ -214,7 +214,7 @@ class Roaring64Map {
         }
         // we put std::numeric_limits<>::max/lowest in parenthesis
         // to avoid a clash with the Windows.h header under Windows
-        return (std::numeric_limits<uint64_t>::max)();
+        return (std::numeric_limits<uint64_t>::lowest)();
```

Comment:
```suggestion
        return (std::numeric_limits<uint64_t>::max)();
```
if bitmap value was empty, we should return limit::max

Reply:
Why function minimum return max value by default ?

Reply:
Shouldn't it return 0 if the bitmap contains no values?

Reply:
We need to distinguish whether the bitmap is empty or not. 
What behaviour do you think is more appropriate. @Dshadowzh

Reply:
In PostgreSQL rb_maximum return 0 if bitmap is empty and rb_minimum return -1 if bitmap is empty. I think we can follow this rule

Reply:
@blackstar-baba  I think this is a good suggestion and we can try following this.
INPUT: NULL OUTPUT: NULL
INPUT: bitmap_empty OUTPUT: 0 or -1

Reply:
Ok, I will modify the code to follow this rule